stanford/cme/db-analytics1.yml

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ansible-playbook
+---
+- include: ../playbooks/db.yml
+  vars:
+    CLUSTER_NUMBER: 1
+    CLUSTER_RDS_NAME: analytics
+    CLUSTER_RDS_PASSWORD: "{{ ANALYTICS_MYSQL_PASSWORD }}"
+    CLUSTER_RDS_USERNAME: "{{ ANALYTICS_MYSQL_USERNAME }}"
+    COMMON_DEPLOYMENT: cme
+  var_files:
+    - "{{ COMMON_DEPLOYMENT_DIR }}/analytics-api.yml"
```

Comment:
For a future enhancement, it may make sense to move most of these vars/files into an intermediary play, so that we aren't duplicating all of these variables multiple times. I imagine it looking like:

# `stanford/playbooks/db-analytics.yml`

```yaml
#!/usr/bin/env ansible-playbook
---
- include: ./db.yml
  vars:
    CLUSTER_NUMBER: 1
    CLUSTER_RDS_NAME: analytics
    CLUSTER_RDS_PASSWORD: "{{ ANALYTICS_MYSQL_PASSWORD }}"
    CLUSTER_RDS_USERNAME: "{{ ANALYTICS_MYSQL_USERNAME }}"
  var_files:
    - "{{ COMMON_DEPLOYMENT_DIR }}/analytics-api.yml"
```

# `stanford/cme/db-analytics1.yml`

```yaml
#!/usr/bin/env ansible-playbook
---
- include: ../playbooks/db-analytics.yml
  vars:
    CLUSTER_NUMBER: 1
    COMMON_DEPLOYMENT: cme
```

# `stanford/cme/db-analytics2.yml`

```yaml
#!/usr/bin/env ansible-playbook
---
- include: ./db-analytics1.yml
  vars:
    CLUSTER_NUMBER: 2
```

Don't sweat it for now, but it could be a future follow-up.